htsget-search/src/htsget/bam.rs

```diff
@@ -12,6 +12,7 @@ use crate::{
   htsget::{Format, HtsGetError, Query, Response, Result, Url},
   storage::{BytesRange, GetOptions, Storage, UrlOptions},
 };
+use crate::htsget::Class;
```

Comment:
could you please move this `Classs` to the previous `htsget::{ ... }` use declaration ?